### PR TITLE
Pull request for branch todo-list that provides basic To-Do List Structure

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Made index.html that provides To-Do list content
- Made style.css file that styles To Do list
- Made index.js file that dynamically creates list items for To-Do List and provides basic input functionality

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+  listItem.classList.add('task');
+  listItem.innerHTML = ` 
+    <input type="checkbox">
+    <label>${toDo.description}</label>
+    <iconify-icon icon="ph:dots-three-outline-vertical-fill" class="dots"></iconify-icon>
+    <hr>  
+  `;
+
+  list.append(listItem);
+});
```
Comment (gemmen29) [inline src/index.js]: - [ ] Kindly add the code that iterates over the tasks inside a function as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Write%20a%20function%20to%20iterate%20over%20the%20tasks%20array%20and%20populate%20an%20HTML%20list%20item%20element%20for%20each%20task.).

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,51 @@
...
+    <iconify-icon icon="bx:refresh"></iconify-icon>
+  </article>
+  <hr>
+  <form>
+    <input type="text" id="new-todo" class="no-outline" placeholder="Add to your list">
+    <button type="submit" id="add-todo" value><iconify-icon icon="uil:enter"></iconify-icon></button>
+  </form>
+  <hr>
+`;
+
```
Comment (gemmen29) [inline src/index.js:39]: - [ ] Kindly sort the array of tasks first before you display it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @jkanyi-web, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @jkanyi-web, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**